app/helpers/reservations_helper.rb

```diff
@@ -42,15 +42,14 @@ def bar_span_positioning_fix
   def manage_reservations_btn # rubocop:disable all
     return if (cannot? :manage, Reservation) || @reservation.reserver.id.nil?
     if (can? :override, :reservation_errors) &&
-       @reservation.approval_status == 'requested'
+       @reservation.flags & Reservation::FLAGS[:request]
```

Comment:
Agreed, we could set up getter and setter methods (`get_flag`, `set_flag`) to make this easier to use / understand in the future.